tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -26,6 +37,79 @@ class TestPVCCreationPerformance(E2ETest):
 
     pvc_size = "1Gi"
 
+    def get_pv_img_count(self, pool_name, interface):
```

Comment:
Please rename the function as get_rbd_or_fs_image_count()

Reply:
fixed in next commit